css-animations-1/animation-shorthand-001.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Animations Test: animation shorthand</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="http://dev.w3.org/csswg/css-animations-1/#animation">
+<meta name="flags" content="animated">
+<meta name="assert" content="Check that animation shorthand property combines four
```

Comment:
Actually there are many combinations of the individual animation properties in the shorthand property.